### ISSUE TITLE
Design and style template for meeting agendas

### ISSUE THREAD

**Issue by jenniferthibault** [18F/fec-style#509]
Open meetings (findable through [the calendar](https://beta.fec.gov/calendar/?category=Open+Meetings)) all have agendas that are put out before the meeting, and are retained afterwards.  These agendas are linked to from the calendar event (and probably elsewhere).

Design a page template in the betaFEC style for these meeting agendas.
- This will probably involve light interaction design improvements, but will primarily be focused on styling.

**Comment by jenniferthibault** [comment]
## Context for getting to these agendas in the wild:

Currently, in the calendar. all links say `Learn more` for both events that have an agenda and those that don't. This could be an opportunity to help clarify that.

**Current:** 
- both say "Learn more" One links to agenda, one links to "[coming soon](http://www.fec.gov/agenda/agenda_generic.shtml)" page
<img width="973" alt="screen shot 2016-09-14 at 2 19 11 pm" src="https://cloud.githubusercontent.com/assets/11636908/18524507/a86a4ef6-7a86-11e6-82c6-fe008bbec52e.png">

**Proposal:**
- If an agenda exists, link directly to it, using specific language.
- If an agenda link does not exist, do not link anywhere else. State that no agenda exists, and do not provide a deeper link.
<img width="960" alt="screen shot 2016-09-14 at 2 24 12 pm" src="https://cloud.githubusercontent.com/assets/11636908/18524580/f3ddf158-7a86-11e6-85cb-79b16f5c8ecc.png">

Dev: @noahmanger is something like this possible? 
Interaction: @nickykrause / @onezerojeremy whaddya think?

## Template design

For the page styling itself, here's my first pass which follows the same structure as the Latest Updates page templates:

![meetingagenda-desktop-v1](https://cloud.githubusercontent.com/assets/11636908/18525101/2616b90a-7a89-11e6-92a6-fa6afc768968.png)

## A couple specific places I'd like feedback:

### Content design:
- I tried my hand at placing some useful information about public meetings that had previously been on the agenda index page at the top of the agenda template.  
- It also seems helpful to tell interested people how to attend this meeting right on that page, so I have placeholder for that.


![screen shot 2016-09-14 at 2 42 04 pm](https://cloud.githubusercontent.com/assets/11636908/18525225/96c2cb94-7a89-11e6-94bd-77221c3bfe37.png)

- I restructured the way some of the elements are styled. This is for visual information flow purposes, but also overlaps content design

 - **Before:**
 - <img width="427" alt="screen shot 2016-09-14 at 2 47 28 pm" src="https://cloud.githubusercontent.com/assets/11636908/18525395/489c656e-7a8a-11e6-9b3e-38b4a194db5f.png">

 - **Proposed update:**
 - ![screen shot 2016-09-14 at 2 47 23 pm](https://cloud.githubusercontent.com/assets/11636908/18525553/fad0ac18-7a8a-11e6-8abd-e56b9e7fe904.png)



@emileighoutlaw what do you think?


### UX design

A big open question for me is what kind of information would be most useful for someone to find from an agenda like this. I've proposed the most recent draft & approved minutes, as well as a card that would link to the calendar, filtered to meetings. 

![screen shot 2016-09-14 at 2 49 30 pm](https://cloud.githubusercontent.com/assets/11636908/18525436/7a503aa4-7a8a-11e6-8bb2-1750b1a8bef0.png)

@onezerojeremy / @nickykrause what do you think?
I know (probably?) we don't have data here, but would this be a workable first draft, or is there a different concept you'd like to try?

**Comment by jenniferthibault** [comment]
I forgot one piece of content design:

With the new content up front, it felt like there was a header missing to start off the actual agenda. I added `Items for discussion:` to help fill this transition, but would love @emileighoutlaw 's thoughts

![screen shot 2016-09-14 at 3 43 37 pm](https://cloud.githubusercontent.com/assets/11636908/18527276/16f0d5ba-7a92-11e6-924a-6a9a04a55288.png)

**Comment by nickykrause** [comment]
@jenniferthibault 

First, I am so glad that "Learn More" will become something more concrete: "View Agenda." +100 to that and to noting that there is no agenda, if no file exists.

Second, I don't have any additional information about what a person would be interested in reading after reading a meeting agenda, but I think that what you've chosen is logical in the abstract. Viewing the draft and approved notes from a previous session would be useful in multiple ways, I imagine...for example, (a) trying to remember what was discussed last time, for continuity, or (b) reviewing an example of the types of things discussed in meetings like these, for those who are new to them.

At first glance, I can't think of something else I would try instead, so it seems like a workable first draft to me.

**Comment by emileighoutlaw** [comment]
I just did some content auditing of agendas, and my take is:

1. There are no well-written agendas in the world (no surprise)
2. You're on a great path already (no surprise) 

I like your proposed edits to how visual elements are styled. It clarifies the parent-child relationship, and I think it makes it easier to read.

As a header, I'm crazy about "Items for discussion." Succinct, clear, to the point.


Let me know if I can do anything more helpful than give you all my 👍

**Comment by nickykrause** [comment]
> There are no well-written agendas in the world (no surprise)

😆

 > You're on a great path already (no surprise)

💯

**Comment by jenniferthibault** [comment]
Heh, thanks all. 

KK, last bits to wrap this up & move it along.

- @emileighoutlaw do you want to hunt down/write the "how to attend this meeting" info and copy, or should I give it a go?  
- @noahmanger how possible is it to do the changes to the "Learn more" links to the agendas on the calendar?
Also @noahmanger : How would you like to integrate this work with the PR in https://github.com/18F/fec-cms/pull/490 ?

**Comment by johnnyporkchops** [comment]
If at some point you want to find out more about user possible user-stories for the agenda pages, Shawn Woodhead Johnson, the Commission Secretary would be a good person to query, but it looks like you've got it pretty well figured out now.

**Comment by emileighoutlaw** [comment]
I'm :face-palm: right now, because I completely missed the section where you kindly and specifically asked for content design help. Abject apologies, @jenniferthibault . 
 
### Open meetings agenda
> Open meetings are public meetings in which the Commissioners adopt new regulations, issue advisory opinions, approve audit reports concerning political committees. Open meetings typically are held twice a month, at 10:00 a.m. on Thursdays.

>Members of the public can attend an open meeting or hearing at the FEC. Please bring a photo ID and be prepared to go through a security check.  You will then be escorted to the Commission's hearing room on the ninth floor. 

This is what we're using in the Press section, so it seems nice from a consistency standpoint, if it looks good to you.

**Comment by jenniferthibault** [comment]
Here's an updated screenshot with that new content in place. I also noticed one small formatting change that could bring the date and time position more in line with existing press templates. 

@johnnyporkchops - is this an ok adjustment to make in your pull request?

<img width="852" alt="screen shot 2016-09-27 at 11 55 55 am" src="https://cloud.githubusercontent.com/assets/11636908/18883271/91a8c16e-84b0-11e6-8018-01a3d84699cb.png">

I was chatting with @johnnyporkchops as he's been working on https://github.com/18F/fec-cms/pull/490 to implement this, and we wanted to raise a content formatting question.
@emileighoutlaw are you interested in opening a conversation about moving from roman numerals to another format (Arabic numerals 1, 2, 3, or other?) based on any best practices or readability research you know of? This popped up because using roman numerals prompted an addition to the `fec-style` repo, because out of the box, it defaults to Arabic numerals. So, there is sort of a dev advantage to not using roman. If there are also readability or stylistic advantages, we may want to open the conversation.

**Comment by emileighoutlaw** [comment]
I am interested in discussing a move to Arabic numerals— I have a hunch Arabic numerals are easier for readers to scan and parse.

In an initial hunt, I had trouble tracking down statistics that support my hunch. That said, I did find many articles explaining how to read Roman numerals (and people asking how to read them) and none explaining Arabic numerals. That in itself seems to be a good case for Arabic.

I'd like to tag in @AmyKort here— I wonder who actually writes the agendas and if you have any gut sense about how contentious a move to Arabic numerals would be?

**Comment by AmyKort** [comment]
The agendas are generated by our Office of the Commission Secretary. I'm happy to see what folks think about numerals, Arabic and otherwise.

**Comment by jenniferthibault** [comment]
I should probably say that if there **aren't** readability or stylistic advantages from a content strategy perspective, it's a pretty small change to `fec-style`, so 👍 to leave them as is: roman numerals.

**Comment by emileighoutlaw** [comment]
I think there is a small-medium content strategy advantage. But nothing worth having a big struggle over. So we'll let Amy check temperatures and go from there. <3 to Amy, as always

**Comment by emileighoutlaw** [comment]
p.s. We're using arabic numerals in other sections of our site, so there are also consistency advantages. But again, these are small to medium :)

**Comment by johnnyporkchops** [comment]
I think the argument that we are already using arabic has a lot of weight. 
I tend to think Roman is just a stylistic thing that  makes sense if the sub-items and their sub-items have numbering, but ours agendas' do not. I ran it by the person who is the content owner (for all intents and purposes) of these pages and she has no problem with changing it if we want to, but I know Amy is still to get back to us.

**Comment by johnnyporkchops** [comment]
So just to confirm, we have the go-ahead to use Arabic numbering for the agenda pages. I make update the PR for that. Good news: no changes to fec-style at this time.

**Comment by AmyKort** [comment]
Also, @johnnyporkchops is awesome. He forgot to mention that in his comment above. :)

**Comment by jenniferthibault** [comment]
Nice work everyone!

_Bonus fun facts:_
This sent me into a research rabbit hole to find out why I was calling them "Arabic numerals" . Turns out the full, complete, historically accurate name would be "[Hindu-Arabic numerals](https://www.unc.edu/~rowlett/units/roman.html)" since they were introduced by those mathematic placement systems. But seems like people also call them "Modern" numerals. 

So there we go. Modern / Hindu-Arabic numerals it is! Thank you @johnnyporkchops . I'm going to close out this issue, since all design questions are resolved. 

Officially moving to implementation and wrapping up in https://github.com/18F/fec-cms/pull/490

**Comment by emileighoutlaw** [comment]
That  _is_ fascinating! Thanks for all the hard work @johnnyporkchops !

**Comment by johnnyporkchops** [comment]
In terms of the Hindu-Numeric numerals, as I realized after we closed this: CSS would default to this style for an OL normally, but most global or reset stylesheets will reset that to list-style-type:none, so if I can find an instance where we are presently using a class to set that to Hindu-Arabic numerals, then I an reuse that style. I could not find one where the numbering was rendered via a list-style-type declaration. Does anyone know where on the beta this exists?
